e2e/playwright/regression-tests.spec.ts

```diff
@@ -590,8 +590,8 @@ extrude002 = extrude(profile002, length = 150)`
     const u = await getUtils(page)
 
     // Constants and locators
-    const planeColor: [number, number, number] = [170, 220, 170]
-    const bgColor: [number, number, number] = TEST_COLORS.DARK_MODE_BKGD
+    const planeColor: [number, number, number] = [80, 60, 60]
+    const bgColor: [number, number, number] = [30, 30, 30]
```

Comment:
This bgColor thing should be reverted but can't afford to wait for another CI run, can do later